pages/common/find.md

```diff
@@ -10,30 +10,26 @@
 
 `find {{root_path}} -name '{{*pattern_1*}}' -or -name '{{*pattern_2*}}'`
 
-- Find directories matching a given name:
+- Find directories matching a given name, in case-insensitive mode:
 
-`find {{root_path}} -type d -name {{*lib*}}`
+`find {{root_path}} -type d -iname {{*lib*}}`
 
-- Find files matching path pattern:
+- Find files matching a path pattern:
 
 `find {{root_path}} -path '{{**/lib/**/*.ext}}'`
 
-- Run a command for each file, use {} within the command to access the filename:
+- Find files matching a given pattern, excluding specific paths:
 
-`find {{root_path}} -name '{{*.ext}}' -exec {{wc -l {} }}\;`
-
-- Find files modified in the last 24-hour period:
-
-`find {{root_path}} -mtime {{-1}}`
+`find {{root_path}} -name '{{*.py}}' -not -path '{{*/site-packages/*}}'`
 
-- Find files using case insensitive name matching, of a certain size:
+- Find files of a certain size:
```

Comment:
I would suggest rewording this to better clarify that those are lower and upper limits and not exact sizes.

Reply:
Yeah, now that the size example is not overloaded with the case-insensitive variation, the description can be a bit more explicit. 👍

Will change.

Reply:
Done!